tests/Unit/Emogrifier/HtmlProcessor/HtmlNormalizerTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Pelago\Tests\Unit\Emogrifier\HtmlProcessor;
+
+use Pelago\Emogrifier\HtmlProcessor\AbstractHtmlProcessor;
+use Pelago\Emogrifier\HtmlProcessor\HtmlNormalizer;
+use Pelago\Tests\Unit\Emogrifier\HtmlProcessor\Fixtures\TestingHtmlProcessor;
+
+/**
+ * Test case.
+ *
+ * @author Oliver Klee <github@oliverklee.de>
+ */
+class HtmlNormalizerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     */
+    public function classIsAbstractHtmlProcessor()
+    {
+        static::assertInstanceOf(AbstractHtmlProcessor::class, new HtmlNormalizer('<html></html>'));
+    }
+
+    /**
+     * @test
+     */
+    public function reformatsHtml()
+    {
+        $rawHtml = '<!DOCTYPE HTML>' .
+            '<html>' .
+            '<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>' .
+            '<body></body>' .
+            '</html>';
+        $formattedHtml = '<!DOCTYPE HTML>' . "\n" .
+            '<html>' . "\n" .
+            '<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"></head>' . "\n" .
+            '<body></body>' . "\n" .
+            '</html>' . "\n";
+
+        $subject = new TestingHtmlProcessor($rawHtml);
```

Comment:
I think this line should read `$subject = new HtmlNormalizer($rawHtml);`.

Reply:
Oh, right. I'm going to fix this in a minute. Good catch!

Reply:
Fixed and re-pushed.